packages/contracts/tasks/fetch-batches.ts

```diff
@@ -52,28 +56,42 @@ task('fetch-batches')
         const tx = await provider.getTransaction(event.transactionHash)
         const batch = (SequencerBatch as any).fromHex(tx.data)
 
-        // Add an extra field to the resulting json
-        // so that the serialization sizes can be observed
+        // Add extra fields to the resulting json
+        // so that the serialization sizes and gas usage can be observed
         const json = batch.toJSON()
         json.sizes = {
           legacy: 0,
           zlib: 0,
         }
+        json.gasUsage = {
+          legacy: 0,
+          zlib: 0,
+        }
 
         // Create a copy of the batch to serialize in
         // the alternative format
         const copy = (SequencerBatch as any).fromHex(tx.data)
```

Comment:
`as any` is required here because there is no `@types/bufio` package - the types live in `@eth-optimism/core-utils/@types`, perhaps we can add that to the `tsconfig.js` and have it pick it up? Ideally the types get published, its just something that I haven't gotten around to